ROOTFS/etc/waggle/sanity/fatal/psu_status.test

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+# A test to ensure proper PSU status
+
+
+pf="PSU Status test"
+echo "$pf: Begin"
+
+acok=$(waggle-get-config -s wagman -k gpio-acok3)
+pg1=$(waggle-get-config -s wagman -k gpio-pg1)
+pg2=$(waggle-get-config -s wagman -k gpio-pg2)
+pg3=$(waggle-get-config -s wagman -k gpio-pg3)
+pg4=$(waggle-get-config -s wagman -k gpio-pg4)
+
+gpiopath="/sys/class/gpio"
+
+# not my code, copied from Joe's PSU test: ../interactive/99_psu_control.test
+gpio_high() { 
+
+        gpio=$1
+        if [ ! -f $gpiopath/gpio$gpio/value ]; then 
+                echo "$pf: Cannot find value of device $gpio"
+                exit 1
+        fi
+
+        if [ $(cat $gpiopath/gpio$gpio/value) -eq 1 ]; then 
+                return 0
+        else 
+                return 1
+        fi
+}
+verify_export() { 
+
+        gpio=$1
+
+        # try to export, it will give us error if we can't
+        if [ ! -d $gpiopath/gpio$acok ]; then 
+                echo $gpio > $gpiopath/export
+
+        fi
+
+        if [ "$?" != 0 ]; then
```

Comment:
You need to move this within the above conditional.  If the above conditional does not run it means it was already exported (right?), so we dont want to test `$?`.